tools/ci/get_version.py

```
@@ -6,67 +6,69 @@
-                    tag = f"{prefix}{patch}-beta.1"
+                    version = f"{stable_match.group(1)}{int(stable_match.group(2)) + 1}-beta.1"
                 else:
-                    tag = "v0.1.0-beta.1"
+                    version = "v0.1.0-beta.1"
 
-        version = tag
         should_push_tag = True
+    else:
+        # PR 或非发布构建
+        short_hash = subprocess.run(
+            ['git', 'rev-parse', '--short', 'HEAD'],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)